sdk/clientcore/core/src/main/java/io/clientcore/core/instrumentation/Instrumentation.java

```diff
@@ -127,15 +167,25 @@ public interface Instrumentation {
      *
      * @param applicationOptions Telemetry collection options provided by the application.
      * @param libraryOptions Library-specific telemetry collection options.
+     * @param serviceEndpoint The service endpoint.
```

Comment:
Any reason the `serviceEndpoint` doesn't exist in `LibraryInstrumentationOptions` or `InstrumentationOptions`?

Reply:
good point. Moved to `LibraryInstrumentationOptions` - it's something users configure on the client builder and we can pass to instrumentation through lib options.